src/dlgProfilePreferences.cpp

```diff
@@ -242,9 +242,9 @@ dlgProfilePreferences::dlgProfilePreferences(QWidget* pF, Host* pHost)
     connect(comboBox_menuBarVisibility, qOverload<int>(&QComboBox::currentIndexChanged), this, &dlgProfilePreferences::slot_changeShowMenuBar);
     connect(comboBox_toolBarVisibility, qOverload<int>(&QComboBox::currentIndexChanged), this, &dlgProfilePreferences::slot_changeShowToolBar);
 
-    // This group of signal/slot connections handle updating *this* instance of
+    // This group of signal/slot connections handles updating *this* instance of
```

Comment:
Are you sure it is appropriate to change the transitive verb to the "*Third-person singular simple present indicative form of handle*"? https://en.wiktionary.org/wiki/handles#English

:neckbeard:

Reply:
Yes. Its subject is "group" which is singular.